src/App.test.js

```diff
@@ -1,8 +1,42 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import App from "./App";
+import Home from "./pages/Home";
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("when app renders", () => {
+  it("renders a header and footer component", () => {
+    //arrange
+    const renderedApp = shallow(<App />);
+    //act
+    const renderedHeader = renderedApp.find("Header");
+    const renderedFooter = renderedApp.find("Footer");
+    // console.log("rendered header: ", renderedHeader);
+    //what the heck is `shallowWrapper {}`
+    //assert
+    expect(renderedHeader.length).toEqual(1);
+    expect(renderedFooter.length).toEqual(1);
+  });
+
+  it("provides a route to home component", () => {
+    //arrange
+    const renderedApp = shallow(<App />);
+    //act
+    const renderedHomeRoute = renderedApp.find('[path="/"]');
+    console.log("renderedHomeRoute debug: ", renderedHomeRoute.debug());
+    console.log("rendered Props: ", renderedHomeRoute.props());
+    console.log("home component: ", Home);
+
+    // const renderedNotFoundRoute = renderedApp.find(
```

Comment:
If this is not being used, it can be removed.

Reply:
thank you!